pep-0624.rst

```diff
@@ -0,0 +1,256 @@
+PEP: 624
+Title: Remove Py_UNICODE encoder APIs
+Author: Inada Naoki <songofacandy@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 07-Jul-2020
```

Comment:
You could keep the original creation date ;-) (I don't ask you to change anything)